website/content/docs/concepts/seal.mdx

```diff
@@ -172,10 +172,9 @@ any storage backend.
    and wait until this process is complete (look for `seal re-wrap completed`).
 
 1. Seal migration is now completed. Take down the old active node, update its
-   configuration of the old active node to use the new seal blocks (completely
-   unaware of the old seal type) and bring it back up. It will be auto-unsealed if
-   the new seal is one of the Auto seals, or will require unseal keys if the new
-   seal is Shamir.
+   configuration to use the new seal blocks (completely unaware of the old seal type)
+   ,and bring it back up. It will be auto-unsealed if the new seal is one of the
```

Comment:
The comma here is missing a space afterwards (or move the comma to be a the EOL above).